static/app/views/insights/mobile/screens/views/screenDetailsPage.tsx

```diff
@@ -61,7 +62,7 @@ export function ScreenDetailsPage() {
     {
       key: 'screen_rendering',
       label: t('Screen Rendering'),
-      alpha: true,
+      featureBadge: 'experimental',
```

Comment:
@narsaynorath Here's another small change I sneaked in: Screen Rendering is now marked as experimental, as `alpha` / `beta` seems to be [reserved for internal / selected customers only](https://github.com/getsentry/sentry/blob/d3f551d5c6254288a404caec1f0146391d904a79/static/app/components/badge/featureBadge.tsx#L29-L30).